### PR TITLE
CliTester: Nice error when cannot run PHP interpreter

### PR DESCRIPTION
**Pull request by xmedeko**
- feature
- BC break - no

### CODE REVIEW

File: src/Runner/CliTester.php [hunk 1]
```diff
@@ -161,6 +161,9 @@ private function createPhpInterpreter()
 			NULL,
 			array('bypass_shell' => TRUE)
 		);
+		if ($proc === false) {
```
Comment (dg) [inline src/Runner/CliTester.php]: Use please `if ($proc === FALSE)` or `if (!$proc)`.

File: src/Runner/CliTester.php [hunk 2]
```diff
@@ -161,6 +161,9 @@ private function createPhpInterpreter()
...
+			throw new \Exception('Cannot run php interpreter ' . $this->options['-p'] . '. Use -p option.');
```
Comment (dg) [inline src/Runner/CliTester.php]: PHP should be uppercased

File: src/Runner/CliTester.php [hunk 3]
```diff
@@ -161,6 +161,9 @@ private function createPhpInterpreter()
...
+		};
```
Comment (Majkl578) [inline src/Runner/CliTester.php]: Extra semicolon.

**Comment by xmedeko** [comment]
Hi guys, I don't know how to corrent the pull request since I've deleted my fork already. I have restored the fork from pull request commit and corrected it, see https://github.com/xmedeko/tester/commit/3a36426099ac3ae1385b5c83556f9317e1ae2002 or squashed into one correct commit see https://github.com/xmedeko/tester/commit/45a884e274db5af6640afd383a5c342f21ba501a

**Pull request closed without merging**